docs/ref/forms/api.txt

```diff
@@ -260,9 +260,16 @@ precedence::
 
 .. method:: Form.get_initial_for_field(field, field_name)
 
-Use :meth:`~Form.get_initial_for_field()` to retrieve initial data for a form
-field. It retrieves data from :attr:`Form.initial` and :attr:`Field.initial`,
-in that order, and evaluates any callable initial values.
+Returns the initial data for a form field. The value is retrieved from
+:attr:`Form.initial`, or :attr:`Field.initial` if not present, and then
```

Comment:
Maybe chop 'then'
Maybe chop 'if not present'

Reply:
Thanks, @smithdc1. On this one, I think it's important to say when `Field.initial` is used. Do you have alternative wording that includes that information? On the subjective points, I'll wait until @carltongibson weighs in before updating the PR.

Reply:
Yes, finding the _right_ words is tricky.  

The other thought I've just had is that we should bear in mind the work of the translators. If we can't find something that is much better then maybe we should not change this paragraph? 

Anyhow, my attempt would be something like.

It tries to retrieve the data firstly from form.initial and secondly from field.initial.

Reply:
On this, I decided to go with the approach used in CPython's [`dict.get()`](https://docs.python.org/3/library/stdtypes.html#dict.get) docs (substituting "else" -> "otherwise"). I still think the conditional aspect is important to communicate (`Field.initial` is consulted only _if_ it's not present in `Form.initial`), which is something the `dict.get()` docs do.

Reply:
The new wording is great. 👍

Reply:
Thanks, @smithdc1! 🙏